app/styles/styles.scss

```diff
@@ -1,7 +1,7 @@
 @forward "uswds-theme.scss";
+@forward "globals.scss";
```

Comment:
We'll want to keep this below the `@forward "usa-button"` import. The [USWDS style guide](https://github.com/uswds/uswds?tab=readme-ov-file#sass-and-theme-settings) recommends it comes second and all other custom styles follow it.

Reply:
👍 Fixed.